sample-data/plugin/configuration.packages.json

```diff
@@ -2,7 +2,7 @@
   "packages": {
     "sk": {
       "PackageName": "EdFi.Ods.Extensions.Sk",
-      "PackageVersion": "1.0.46",
+      "PackageVersion": "1.0.150",
```

Comment:
@semalaiappan https://github.com/Ed-Fi-Alliance-OSS/Starter-Kit-SEA-Data-Collection/blob/main/packer/scripts/sk.ps1#L15 needs to be updated as well. Currently this package version is in two places.

Reply:
Yeah, this is looking good at load time and is now correctly running the script as expected.  Let's get it caught up with latest and do the update Vinaya mentioned and then we should be good to get it in.